docs/pinecone-quickstart.ipynb

```diff
@@ -62,68 +60,49 @@
       "source": [
         "## Sign up or log in\n",
         "\n",
-        "Sign up for a [free Pinecone account](https://www.pinecone.io/pricing/) and get an API key.\n",
-        "\n",
-        "If you already have an account, this step logs you in and gets your API key.\n"
+        "Sign up for a [free Pinecone account](https://www.pinecone.io/pricing/) and get an API key. Put your key in `PINECONE_API_KEY` below when initializing a new client instance:\n"
```

Comment:
@jamescalam, can we find a way to allow this connect widget in notebooks? It's a new feature that makes the Pinecone auth flow a lot quicker/easier. 

cc @tejchilli